installparty.sh

```diff
@@ -49,10 +49,12 @@ source ~/.bashrc
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
 
-nvm -v 
+nvm -v
 nvm install stable
 nvm use stable
 
```

Comment:
yeah maybe. but just in case they have something they wanna keep in their dotfiles, i think it'll be safer to echo or cat it in.